### ISSUE TITLE
Hyperparameters (lambda) of the loss function

### ISSUE THREAD

**Issue by preetom-saha-arko** [Kartik-3004/facexformer#24]
<img width="946" height="103" alt="Image" src="https://github.com/user-attachments/assets/bc5781ab-9659-436f-8277-c0420ea629c7" />

The paper mentioned this loss function but nowhere mentioned the values of lambdas. May I know the values of all the lambdas used in the loss function?
